src/chrome/content/rules/My-aime.net.xml

```diff
@@ -0,0 +1,11 @@
+<!--
+	Timeout:
+		- mx
+-->
+<ruleset name="my-aime.net">
```

Comment:
```suggestion
<ruleset name="My-aime.net">
```

To match the file name and `ruleset` `name`.

Reply:
done (capitalized the A as well)